Index: Header.tpl
===================================================================
--- Header.tpl	(revision 1419)
+++ Header.tpl	(working copy)
@@ -49,7 +49,7 @@
 	<!-- END -->
 	<script language="JavaScript" type="text/javascript" id="_current_language_" src="include/js/{php} echo $_SESSION['authenticated_user_language'];{/php}.lang.js?{php} echo $_SESSION['vtiger_version'];{/php}"></script>
 	<script language="JavaScript" type="text/javascript" src="include/js/QuickCreate.js"></script>
-	<script language="javascript" type="text/javascript" src="include/scriptaculous/prototype.js"></script>
+	{if $MODULE_NAME neq 'MarketingDashboard'}<script language="javascript" type="text/javascript" src="include/scriptaculous/prototype.js"></script>{/if}
 	<script language="JavaScript" type="text/javascript" src="include/js/menu.js?v={$VERSION}"></script>
 	<script language="JavaScript" type="text/javascript" src="include/calculator/calc.js"></script>
 	<script language="JavaScript" type="text/javascript" src="modules/Calendar/script.js"></script>
